src/chainparamsbase.h

```diff
@@ -40,9 +43,9 @@ class CBaseChainParams
 std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain);
 
 /**
- *Set the arguments for chainparams
+ * @return true if BaseParams have been selected, false if not
  */
-void SetupChainParamsBaseOptions();
+bool BaseParamsSelected();
```

Comment:
Nit: This can be `const`, I assume.

Reply:
Assume you mean `const` member, but it's not a member function, so no object to be `const` on.

Reply:
Oh yes indeed - I somehow thought this was a member.